### PR TITLE
[WIP] Convert localized decimal notation

### PR DESCRIPTION
**Pull request by james-prado**
Closes https://github.com/MyEtherWallet/MyEtherWallet/issues/335
* Use [parse=decimal-number](https://github.com/AndreasPizsa/parse-decimal-number) to convert localized number strings into float values (`1.234.567,89` => `1234567.89`)
* Update `<UnitConverter />`, allowing it to parse localized strings too

* Todo: 
    * Update `amount` inputs in swap tab

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "idna-uts46": "1.1.0",
     "lodash": "4.17.4",
     "moment": "2.19.2",
+    "parse-decimal-number": "^1.0.0",
```
Comment (wbobeirne) [inline package.json]: Lock that version down

File: common/components/renderCbs/UnitConverter.tsx [hunk 2]
```diff
@@ -49,7 +49,13 @@ export class UnitConverter extends Component<Props, State> {
     });
   }
   private baseUnitCb = (value: string, decimal: number) => {
-    const baseUnit = toTokenBase(value, decimal).toString();
+    console.log('original value', value);
```
Comment (HenryNguyen5) [inline common/components/renderCbs/UnitConverter.tsx]: Got a lot of console logging here

File: common/components/renderCbs/UnitConverter.tsx [hunk 3]
```diff
@@ -49,7 +49,13 @@ export class UnitConverter extends Component<Props, State> {
...
+    const parsedValue = parseDecimalNumber(value, '.,')
+      ? parseDecimalNumber(value, '.,').toString()
```
Comment (HenryNguyen5) [inline common/components/renderCbs/UnitConverter.tsx]: You can avoid the double `toString()` calls here by calling `toString()` once on line 57

**Comment by dternyak** [comment]
Closing for now until we circle back to this issue.

**Pull request closed without merging**